### ISSUE TITLE
Error on VS2019

### ISSUE THREAD

**Issue by JBaltika** [uli-weltersbach/XPathTools#41]
Hi,
I am getting quite strange error a lot in VS2019 ... Any  thoughts?

  <entry>
    <record>2894</record>
    <time>2019/07/26 14:31:56.795</time>
    <type>Error</type>
    <source>Editor or Editor Extension</source>
    <description>System.ArgumentException: The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))&#x000D;&#x000A;   at EnvDTE._DTE.get_ActiveDocument()&#x000D;&#x000A;   at ReasonCodeExample.XPathTools.ActiveDocument.get_Current() in C:\projects\xpathinformation\ReasonCodeExample.XPathTools\ActiveDocument.cs:line 14&#x000D;&#x000A;   at ReasonCodeExample.XPathTools.VisualStudioIntegration.XmlTextViewCreationListener.TextViewCreated(IWpfTextView textView) in C:\projects\xpathinformation\ReasonCodeExample.XPathTools\VisualStudioIntegration\XmlTextViewCreationListener.cs:line 38&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.Editor.Implementation.WpfTextView.&lt;&gt;c__DisplayClass254_0.&lt;BindContentTypeSpecificAssets&gt;b__1()&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.CallExtensionPoint(Object errorSource, Action call)</description>
  </entry>

**Comment by uli-weltersbach** [comment]
Could you provide some steps to reproduce the error?

**Comment by AlexN78** [comment]
I have the same error on 1st opening *.csporj file (TargetFramework=netcoreapp3.0) in VS2019 in **preview mode (on preview tab)**. I do not have this error if I open the same file in a new tab (Edit Project File).

**Comment by uli-weltersbach** [comment]
@AlexN78 Could you attach/post the contents of the `csproj`-file?

**Comment by uli-weltersbach** [comment]
Steps to reproduce:
1. Open VS 2019
2. Create class library project for e.g. NetStandard 2.0
3. Right-click the class library project
4. Select "Edit Project File"
5. The following error occurs on first load (as reported by @AlexN78):
```xml
  <entry>
    <record>2036</record>
    <time>2019/08/28 18:39:00.176</time>
    <type>Error</type>
    <source>Editor or Editor Extension</source>
    <description>System.ArgumentException: The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))&#x000D;&#x000A;   at EnvDTE._DTE.get_ActiveDocument()&#x000D;&#x000A;   at ReasonCodeExample.XPathTools.ActiveDocument.get_Current() in C:\projects\xpathinformation\ReasonCodeExample.XPathTools\ActiveDocument.cs:line 14&#x000D;&#x000A;   at ReasonCodeExample.XPathTools.VisualStudioIntegration.XmlTextViewCreationListener.TextViewCreated(IWpfTextView textView) in C:\projects\xpathinformation\ReasonCodeExample.XPathTools\VisualStudioIntegration\XmlTextViewCreationListener.cs:line 38&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.Editor.Implementation.WpfTextView.&lt;&gt;c__DisplayClass254_0.&lt;BindContentTypeSpecificAssets&gt;b__1()&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.CallExtensionPoint(Object errorSource, Action call)&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)</description>
  </entry>
```

**Comment by uli-weltersbach** [comment]
This should be fixed in v6.0.3.
Please reopen this issue if the error persists.
